rb/spec/tests.bzl

```diff
@@ -0,0 +1,85 @@
+load("@rules_ruby//ruby:defs.bzl", "rb_test")
+load("//common:browsers.bzl", "chrome_data")
+
+ENV = select({
+    "//rb/spec/integration:chrome": {
```

Comment:
If I'm reading this right, you'll only be able to test one browser in a single build. Is that correct, or have I missed something?

Reply:
`+` only works for lists, while `|` works for dicts
https://github.com/bazelbuild/starlark/issues/14

Reply:
Yes, at the moment. However, I'd like to add Java-like targets for each browser in the following PRs:

- //rb/spec/integration/selenium/webdriver:driver-chrome
- //rb/spec/integration/selenium/webdriver:driver-firefox
- //rb/spec/integration/selenium/webdriver:driver-edge
- //rb/spec/integration/selenium/webdriver:driver-chrome-remote
- //rb/spec/integration/selenium/webdriver:driver-firefox-remote
- etc